### ISSUE TITLE
consolidated hardware Stabilizer+Pounder

### ISSUE THREAD

**Issue by jordens** [sinara-hw/Pounder#112]
The genericism of Stabilizer is a limitation to the performance when using Pounder (input stage limited in slew rate, distortion noise to achieve flexibility, layout is tight and complex/high viscosity, component choice trade-offs, software/assembly/debugging complexity).

I'd like to consider a single board hardware variant of Stabilizer+Pounder with the following deliberations:

* Single board (debugability, reduced complexity of mechanics, software etc)
* Use CPU's ADC and DACs, no external ADCs, DACs
* no PGIA, merged IF amp and ADC gain+shifting stages
* no EEM/LVDS interface
* design and layout revised for 70 dB RF isolation (input-output and between channels, i.e. at the level of the DDS)
* potentially a variant with an FPGA as coprocessor: then it would be a consolidated version of FastServo+Pounder, i.e. Zynq again with external ADC

**Comment by gkasprow** [comment]
good idea. I don't like these kinds of combo due to ground loop issues
